sns/scripts/proposals/register_canisters_with_sns/translations.ini

```diff
@@ -0,0 +1,4 @@
+TITLE="Register 'Translations' as an SNS controlled canister"
+// TODO: This needs updating once the initial PR is merged
+URL="https://github.com/open-chat-labs/open-chat/blob/8fb174b3ca93ed7c2c8f7df71b613b4c06e81205/backend/canisters/translations/impl/src/lib.rs"
```

Comment:
I reckon all of these canister urls should point to `master` rather than a specific commit.

Reply:
Well it is the so-called permanent url

Reply:
Yes I guess. But the url provided doesn't work and we don't know what commit to put since we won't know the commit Id until this PR is merged.